plugins/catalog-backend-module-aws/src/lib/defaultTransformers.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { EKSClusterEntityGenerator } from '../processors/types';
+import { type Cluster } from '@aws-sdk/client-eks';
+import {
+  ANNOTATION_KUBERNETES_API_SERVER,
+  ANNOTATION_KUBERNETES_API_SERVER_CA,
+  ANNOTATION_KUBERNETES_AUTH_PROVIDER,
+} from '@backstage/plugin-kubernetes-common';
+
+const ACCOUNTID_ANNOTATION: string = 'amazonaws.com/account-id';
+const ARN_ANNOTATION: string = 'amazonaws.com/arn';
```

Comment:
It might make sense to expose some or all of these. They can be handy for people who want to consume those or people writing their own transformer or extension of the default one.

At that point, slightly adjusted names might be reasonable.

Also, prefer common prefixes (see https://github.com/backstage/backstage/blob/master/STYLE.md, API Design, 8.)


```suggestion
const ANNOTATION_AWS_ACCOUNT_ID: string = 'amazonaws.com/account-id';
const ANNOTATION_AWS_ARN: string = 'amazonaws.com/arn';
```